tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,485 @@
+
+    result_header_signature = json.loads(result.output).get('result')[0].get('header_signature')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_header_signature == result_header_signature
+
+
+def test_get_list_batches_with_non_existing_ids():
+    """
+    Case: get a list of batches by non existing identifiers.
+    Expect: list of batches not found error message is returned.
+    """
+    batch_ids = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+                '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8752, ' \
+                '257353cb1180bdce4e19f290e4bfdb48212744a8ae08f6fc974e8c7c2cfe4692' \
+                '19d257a7c62197bd614b9dc13cd562c6612aee45ab0bccfcdd6b0d1ed0b3cdce'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--ids',
+        batch_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': f'List of batches not found.',
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_invalid_ids():
+    """
+    Case: get a list of batches by invalid identifiers.
+    Expect: the following identifier is invalid error message.
+    """
+    invalid_id = '6f200'
+    batch_ids = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+                '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751, ' \
+                f'{invalid_id}'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--ids',
+        batch_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'ids': [
+                f'The following identifier `{invalid_id}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_start():
+    """
+    Case: get a list of batches by batch identifier starting from.
+    Expect: batches are returned.
+    """
+    start = 'fa2d1a209ad04fd2ad7fb5183976e647cc47b4c08e2e578097afc2566a0284e7' \
+            '60eb3f2ff8f72f290765211d4da3341f23091cc7a16805025a17c04a90818a44'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--start',
+        start,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_with_non_existing_start():
+    """
+    Case: get a list of batches by non-existing batch identifier starting from.
+    Expect: list of batches not found error message is returned.
```

Comment:
`list of batches not found error message is returned` > `list of batches not found error message`.

Reply:
Done.